includes/Core/Authentication/Authentication.php

```diff
@@ -1279,4 +1279,29 @@ private function verify_user_input_settings() {
 		}
 	}
 
+	/**
+	 * Filters feature flags using features received from the proxy server.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param boolean $feature_enabled Original value of the feature.
+	 * @param string  $feature_name    Feature name.
+	 * @return boolean State flag from the proxy server if it is available, otherwise the original value.
+	 */
+	private function filter_features_via_proxy( $feature_enabled, $feature_name ) {
+		$transient_name = 'googlesitekit_remote_features';
+		$features       = $this->transients->get( $transient_name );
+		if ( empty( $features ) ) {
```

Comment:
When features are empty (as is currently the case), this will store an empty array in the transient, which will pass the empty check, causing the plugin to check the endpoint repeatedly.

When the transient is expired, get_transient will return false, let's check for that explicitly here.

```suggestion
		if ( false === $features ) {
```

Reply:
Updated.